build/compile.sh

```diff
@@ -258,18 +268,36 @@ python-sources() {
   echo "$OVM_LIBRARY_OBJS" | add-py27
 }
 
-_headers() {
-  local c_module_srcs=${1:-_tmp/hello/c-module-srcs.txt}
-  local abs_c_module_srcs=$PWD/$c_module_srcs
-
+_gcc_extract_headers() {
   # -MM: no system headers
-  cd $PY27
+  local abs_c_module_srcs=$1
   gcc \
     "${INCLUDE_PATHS[@]}" \
     "${PREPROC_FLAGS[@]}" \
     -MM $OVM_LIBRARY_OBJS \
     Modules/ovm.c \
-    $(cat $abs_c_module_srcs) 
+    $(cat $abs_c_module_srcs)
+}
+
+_headers() {
+  local c_module_srcs=${1:-_tmp/hello/c-module-srcs.txt}
+  local abs_c_module_srcs=$PWD/$c_module_srcs
+
+  cd $PY27
+
+  # HACK: We extract Python headers twice, once with a Linux pyconfig.h and
```

Comment:
Hm can you show the difference between these two sets of headers?  I feel like OS X should just use `posixmodule.c` too, and the headers shouldn't be very different, or at all.

Reply:
the difference ended up being fairly minimal as you suspected!